pydoover/cloud/reports/config.py

```diff
@@ -0,0 +1,31 @@
+from ...config import String, Array, Enum
+import zoneinfo
+
+class DevicesConfig(Array):
```

Comment:
I was impatient for this so I copy pasted what you've done into the branch you're targetting but in the `pydoover.config` package instead since I can see us using Devices in more than just reports :)